catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -48,9 +51,12 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(QueryApplication.class);
 
   private static final String APPLICATION_JSON = "application/json";
+  private static final String TRANSFORMER_ID_PROPERTY = "id";
 
   private FeatureService featureService;
 
+  private List<ServiceReference> queryResponseTransformers;
```

Comment:
❓ It looks like only the list of string ids is used, why not move the stream to the setter and eliminate most of the logic from the get?